SU-HSDO/suhumsci
# STN-345: Vertical Linked Card

# READY FOR REVIEW

## Summary
This PR adds the traditional styles for the vertical linked card. Note: the arrow for this component is incorrect but the correct asset is being added in #585.

**Designs:** https://www.figma.com/file/bG42fKKioEcnJ0ORKkLL5E/%E2%9A%99%EF%B8%8F-Traditional-Theme---Working?node-id=5686%3A3

✅ Design review

**Browser tested in:**
- Chrome
- Firefox
- Safari
- Edge
- IE 11

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. Run `npm test`
2. Confirm that the [vertical linked card](http://economics.suhumsci.loc/qa/qa-cards) in the colorful theme is unchanged.
3. Update to the traditional theme and confirm that the vertical linked cards match the design in Figma.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
